.config/semgrep.yaml

```diff
@@ -64,6 +64,9 @@ rules:
   message: "`tokio::spawn` won't set task name by default. use `fedimint_core::runtime::spawn` instead"
   pattern: tokio::spawn
   severity: WARNING
+  paths:
```

Comment:
Why?

Reply:
Oh.

You can use a `// nosemgrep: <rule-name>` to just exclude specific places that need to break the rule.

Reply:
done